### PR TITLE
add carthage support

### PR DESCRIPTION
**Pull request by JuGoo**
you can try to: 
github "JuGoo/SwiftMonkey" "1.0.2"

after merge:
github "zalando/SwiftMonkey"

### CODE REVIEW

File: SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -183,7 +197,8 @@
 		OBJ_20 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				ENABLE_TESTABILITY = YES;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+				ENABLE_BITCODE = NO;
```
Comment (DagAgren) [inline SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj:201]: This seems questionable. We've had problems with bitcode while using Carthage in the past, and I don't want to repeat that. Is it needed?

Comment (kemchenj) [inline SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj]: Many popular frameworks enable bitcode as default now, like Alamofire, RxSwift etc.

Comment (kemchenj) [reply SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj:201]: Bitcode now works fine. Many popular frameworks enable bitcode as default now, like Alamofire, RxSwift etc.

Comment (DagAgren) [reply SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj:201]: Which is why I am worried about this pull request disabling it.

@JuGoo, can you explain why this change is here?

Comment (kemchenj) [reply SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj:201]: I think I know why. When running UITest in a real device, will raise a error like this:

> `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest` does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64

**Comment by wojciechczerski** [comment]
Fixed by #56. Closing the issue.

**Pull request closed without merging**